frontend/src/container/ExplorerOptions/ExplorerOptions.tsx

```diff
@@ -501,12 +507,18 @@ function ExplorerOptions({
 			setIsRecentlyUsedSavedViewSelected(false);
 		}
 
-		return (): void => clearTimeout(timeoutId);
+		// eslint-disable-next-line consistent-return
```

Comment:
curious why we moved away from the previous eslint compliant code to disabling the rule ?

Reply:
+1